logs_http.module

```diff
@@ -93,146 +68,28 @@ function _logs_http_decode_exception($exception) {
  * devel_boot() registers this function as a shutdown function.
  */
 function logs_http_shutdown() {
-  if (!$events = logs_http_get_registered_events()) {
+  $logs_http_logger = \Drupal::service('logs_http.logs_http_logger');
```

Comment:
you should type hint this, so PHPstorm helps you. (look at my previous comment about this, I've provided an example for the type hint, so it's something like:

```php
/** @var LogsHttpInterface $logs_http_logger */
$logs_http_logger = \Drupal::service('logs_http.logs_http_logger');
```